scout/server/blueprints/cases/static/case_images.js

```diff
@@ -1,4 +1,5 @@
 const WIDTH_BREAKPOINT = 1955  // less will remove one column
+const SINGLE_COL_WIDTH_BREAKPOINT = 1555
```

Comment:
why exacly 1555?

Reply:
It is based on scientifically exact calculation; unfortunately this window is too narrow to hold it, so the rest is black magic. It is based off the static image sizes given to the svg, which in turn depend on the relative chromosome sizes, also possible sidebar sizes. The part with ballparking common resolutions is the most magic; given a certain resolution or range I think we could settle on a formula. As it is, changing them slightly is advised if they misbehave.